src/OpenTelemetry.Contrib.Instrumentation.Quartz/OpenTelemetry.Contrib.Instrumentation.Quartz.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>OpenTelemetry Quartz.NET Instrumentation</Description>
+    <PackageTags>$(PackageTags);distributed-tracing</PackageTags>
+    <MinVerTagPrefix>Instrumentation.Quartz-</MinVerTagPrefix>
+    <IncludeSharedInstrumentationSource>true</IncludeSharedInstrumentationSource>
+    <TargetFrameworks>net461;net472;netstandard2.0</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="OpenTelemetry.Api" Version="1.1.0-beta4" />
```

Comment:
can you pick 1.1.0 as 1.1.0 stable has released!

Reply:
updated to 1.1.0 stable